wgsl/index.bs

```diff
@@ -3805,8 +3805,106 @@ shorten vector declarations based on the type or number of elements provided.
 If you want `vec4<f32>` you must provide 4 float values in the constructor.
 
 ## Precedence ## {#precedence}
-
-Issue: (dsinclair) Write out precedence rules. Matches c and glsl rules ....
+<table class='data'>
```

Comment:
Perhaps instead of the inline css throughout, consider adding a class to this table, e.g. left-aligned-first-column, and add a rule like:
```css
table.left-aligned-first-column tr th:nth-child(1),
table.left-aligned-first-column tr td:nth-child(1) {
  text-align: left;
}
```

Reply:
Added a roughly similar styling